app/kudos/views.py

```diff
@@ -324,6 +324,8 @@ def send_3(request):
             primary_email = to_emails['events'][0]
         else:
             print("TODO: no email found.  in the future, we should handle this case better because it's GOING to end up as a support request")
+    if type(primary_email) is list and len(primary_email):
```

Comment:
Can also be written as: `if primary_email and isinstance(primary_email, list):`

Reply:
dbec78ef4edab0ce4e996426e44d847cc6c2d813